Competition2017/tests/competition/subsystems/drive/DriveSubsystemTest.java

```diff
@@ -77,5 +77,29 @@ public void lowPowerInputTankDriveTest(){
         drive.tankDrivePowerMode(0.01, -0.01);
         verifyDriveSetpoints(0.01, -0.01);
     }
-}
+    
+    public void testUpdatePeriodicDataTest(){
```

Comment:
This was deleted in another commit, probably a bad merge left it here, safe to delete again from your branch.